src/uploader.js

```diff
@@ -387,6 +387,11 @@ var uploader = {
         options.$uploadBtn.prop('disabled', true);
         options.$resetBtn.prop('disabled', true);
 
+        var importButton = options.$form[0].querySelector('button');
```

Comment:
When using the `var` keyword, you must reflect the hoisting, so "var on top". But this is for ES5 flavor.

This repo supports ES6, so better to use `const` instead.

```suggestion
        const importButton = options.$form[0].querySelector('button');
```

Reply:
Done. I don't put because in my local environment gives error, but maybe because I don't run bundle and directly edit